layouts/data-playground/single.html

```diff
@@ -2,10 +2,144 @@
 {{ .Title }} &ndash; {{ .Site.Title }}
 {{ end }}
 
+{{ define "header" }}
+<div class="relative bg-[#1D65A6]">
+    {{ partial "site-nav.html" . }}
+</div>
+{{ end }}
+
 {{ define "main" }}
 
 <div class="max-w-7xl mx-auto px-4 relative">
 
-</div>
+    <div class="w-full bg-[#F5F5F5] flex justify-between my-5 place-items-center h-36 px-10">
```

Comment:
Thanks for your suggestion. I believe this part should be centered for the same padding on top/bottom and the same padding on left/right because of the bottom. I left a comment in Figma to notify @eschmier